drop-in/src/main/java/com/adyen/checkout/dropin/SessionDropInResult.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 14/2/2023.
+ */
+
+package com.adyen.checkout.dropin
+
+import com.adyen.checkout.dropin.service.DropInServiceResult
+import com.adyen.checkout.dropin.service.SessionDropInService
+import com.adyen.checkout.sessions.model.SessionPaymentResult
+
+/**
+ * A class that contains the final result of Drop-in.
+ */
+sealed class SessionDropInResult {
+
+    /**
+     * Drop-in was dismissed by the user before it has completed.
+     */
+    class CancelledByUser : SessionDropInResult()
```

Comment:
I guess this should be object

Reply:
It's left like that on purpose, if we ever need to change it to a class to add some parameters it will be a breaking change for merchants